tests/server/core/error-pages.js

```diff
@@ -129,12 +129,37 @@ describe('Templating error pages', function () {
 			retval = ret;
 		});
 
-		renderer.render.firstCall.yield(new Error('bad times'));
+		renderer.render.firstCall.yield(new Error('renderer.render threw some error'));
 
 		assert.strictEqual(renderer.render.callCount, 1);
 		assert.strictEqual(null, retval);
 	});
 
+	it('Should callback with null if error falsy', function () {
+		var errorPages = require(moduleName)(config);
+		var retval = false;
+		var error;
+		errorPages.getPage(error, req, res, function (ret) {
+			retval = ret;
+		});
+
+		assert.strictEqual(null, retval);
+	});
+
+	it('Should render if provided err.status falsy', function () {
+		var errorPages = require(moduleName)(config);
+		var retval = false;
+		error.status = undefined;
+		errorPages.getPage(error, req, res, function (ret) {
+			retval = ret;
+		});
+
+		renderer.render.firstCall.yield(null, 'my error page');
+
+		assert.strictEqual(renderer.render.callCount, 1);
```

Comment:
Just to clarify I mean the proclaim thing we're using says this in the docs `proclaim.equal( actual, expected, [message] )`, however we were just doing `expected, actual` which is the wrong order. Normally the message you see in a failure is the third argument. That's normally the standard with assertion libs.

I quickly checked some other test files in `shunter/tests/server/core/`, and can confirm they do `equal` style assertions correctly according to the docs...it seems to just be `error-pages.js` which is the exception